### PR TITLE
reorganize the collator page into a section with multiple pages

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR reorganizes the collator page into a section with multiple pages. It adds an FAQ page, an in-depth requirements page, and a central place for account management related activities including information on proxy accounts and setting an identity.

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [x] If images have been moved, I have created an additional PR in `moonbeam-docs-cn` to update images to their new paths
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables

### Corresponding PRs

https://github.com/PureStake/moonbeam-mkdocs/pull/65
https://github.com/PureStake/moonbeam-docs-cn/pull/49

### After Translation Requirements

- [x] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [x] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

- Image to be removed from English repo: /images/node-operators/networks/collators/collator-polkadotjs-1.png
- Redirect to be added to Chinese mkdocs.yml: https://github.com/PureStake/moonbeam-mkdocs/pull/65

### CODE REVIEW

File: node-operators/networks/collators/activities.md [hunk 1]
```diff
@@ -1,72 +1,19 @@
...
-
- - Importing an existing (or create a new) H160 account from external wallets or services such as [MetaMask](/tokens/connect/metamask/) and [MathWallet](/tokens/connect/mathwallet/)
- - Create a new H160 account with [Polkadot.js](/tokens/connect/polkadotjs/)
-
-Once you have an H160 account imported to Polkadot.js, you should see it under the **Accounts** tab. Make sure you have your public address at hand (`PUBLIC_KEY`), as it is needed to configure your [full node](/node-operators/networks/run-a-node/overview/) with the collation options.
-
-![Account in Polkadot.js](/images/node-operators/networks/collators/collator-polkadotjs-1.png)
+This guide will take you through the important timings to be aware of when leaving or reducing your self-bond amount, how to join and leave the candidate pool, and adjust your self-bond.
 
 ## Collator Timings {: #collator-timings }
```
Comment (albertov19) [inline node-operators/networks/collators/activities.md:18]: Add a round length to the table, so they can do the math

Comment (albertov19) [reply node-operators/networks/collators/activities.md:18]: I also think the !!!note after this table is too long. I would just only leave the part of value might change in the future, and take most of it out of the note

File: node-operators/networks/collators/overview.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+# Run a Collator on Moonbeam
+
+![Collator Moonbeam Banner](/images/node-operators/networks/collators/collator-banner.png)
+
+## Introduction {: #introduction } 
+
+Collators are members of the network that maintain the parachains they take part in. They run a full node (for both their particular parachain and the relay chain), and they produce the state transition proof for relay chain validators.
+
+There are some [requirements](/node-operators/networks/collators/requirements/){target=blank} that need to be considered before they can become a collator candidate including machine, bonding, and account requirements.
```
Comment (albertov19) [inline node-operators/networks/collators/overview.md:14]: I think here we should a bit of a bullet list of the requirements to be a collator (like a summary of both technical and "moral"):

- Be engaged with the network and community
- Support the livelyness of the network
- .... some other moral and technical requirements
-

**Review by albertov19 — CHANGES_REQUESTED** [review]
Great job! I really liked the whole section!

File: node-operators/networks/collators/overview.md (hunk 2 above)
Comment (eshaben) [reply node-operators/networks/collators/overview.md]: What are the other "moral" requirements? Would it be better to add them to the Requirements page?

Comment (albertov19) [reply node-operators/networks/collators/overview.md]: @artkaseman what "moral" requirements should we list here.

We could add them in the requirements page, but I feel that a short summary in the overview would also be helpful

**Comment by eshaben** [comment]
Merging this for now... will circle back to the "moral" requirements part a little later

**Review by albertov19 — APPROVED** [review]

**Pull request merged**